docs/technical-reference/build-test-run.md

```diff
@@ -290,3 +290,7 @@ In the module settings, add the source folder and test source folders of that mo
 ![Screenshot of module settings in IntelliJ](/img/intellij-module-settings.png)
 
 Then, do the same thing for the main OpenRefine project and now you are good to go.
+
```

Comment:
## Code style in IntelliJ
```suggestion

## Code style in Eclipse

```

Just to keep the structure aligned between the two IDEs